website/templates/profile/notifications.mako

```diff
@@ -44,20 +45,31 @@
                     <!-- Flashed Messages -->
                     <div data-bind="html: message, attr: {class: messageClass}"></div>
             </div><!--view model scope ends -->
-            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
+                <form id="selectNotifications">
+                    <div id="grid"></div>
+                </form>
+        </div>
     </div>
 </div>
 </%def>
 
 <%def name="javascript()">
     <% import website %>
+    <% import json %>
     ${parent.javascript()}
     <script type="text/javascript">
-        window.contextVars = $.extend({}, window.contextVars, {'mailingList': '${website.settings.MAILCHIMP_GENERAL_LIST}'});
+        window.contextVars = $.extend({}, window.contextVars, {'mailingList': '${website.settings.MAILCHIMP_GENERAL_LIST}',
+                                                               'notificationsData': ${json.dumps(notifications_data)}
```

Comment:
Once you've added an endpoint to `GET /api/v/subscriptions/` (see previous comment), this should no longer be necessary.